extension/js/common/browser.ts

```diff
@@ -827,6 +827,10 @@ export class XssSafeFactory {
     return this.frameSrc(this.extUrl('chrome/elements/pgp_pubkey.htm'), { frameId: this.newId(), armoredPubkey, minimized: Boolean(isOutgoind), });
   }
 
+  srcBackupIframe = (armoredPubkey: string, isOutgoind?: boolean) => {
```

Comment:
typo: `isOutgoind` -> `isOutgoing`

But it should be removed anyway, it's not used and not useful for private keys.

It's useful for public keys, because those may be sent over email. If a public key is outgoing, that means I sent it to someone else, then it's a waste of space to print out all the details about it. Because private keys are never outgoing (you would not be sending them to other people), `isOutgoing` is not useful for private key backups.

Please remove, thanks!

Reply:
Got it, Thanks!